scenes/brick.gd

```diff
@@ -0,0 +1,15 @@
+extends StaticBody2D
+
+const simple_brick_scene = preload("res://scenes/power_up.tscn")
+
+func _ready():
+	pass
+
+func collapse_and_maybe_drop_power_up():
```

Comment:
Change `collapse_and_maybe_drop_power_up` ~> `queue_free_with_powerup`.

We should be able to control the probability of the power-up spawning.